### ISSUE TITLE
[Mechanical Concept] Spores

### ISSUE THREAD

**Issue by minichibis** [ProbablyTez/Agony#399]
Spores are much like bullets, except for a few special properties

1. They bounce off walls and have infinite range
2. They can be killed by Isaac
3. They (usually) travel slowly

note that spores cannot close a room.

Various enemy alts in catacombs would release spores on death. Various other enemies could be based off the idea of spores.

**Comment by Floofball** [comment]
Ooh, neat idea! Hope this gets approved. Adding tear effects to enemies is always good for variety.

**Comment by Anonymous-Koala** [comment]
I like this as well. Seems pretty interesting.

**Comment by ProbablyTez** [comment]
Yeah, sounds interesting, good idea putting this here too, since now it's easier for other designign catacomb enemies to use these spores
